Sources/NIOExtras/DebugOutboundEventsHandler.swift

```diff
@@ -0,0 +1,88 @@
+import NIO
+
+public class DebugOutboundEventsHandler: ChannelOutboundHandler {
+    
+    public typealias OutboundIn = ByteBuffer
```

Comment:
This is incorrect.

The correct two typealiases are `OutboundIn = Any` and `OutboundOut = Any`.